docs/references/concepts.md

```diff
@@ -4,16 +4,35 @@ description: Fundamental user-facing concepts
 
 # Key Concepts
 
+## Instance
+
+An "instance" of Opstrace runs inside your cloud account.
+It is installed with the CLI `opstrace create` command.
+This name must be _globally_ unique and limited to 23 characters (`[a-z0-9-_]`) because it used to provide a conveient DNS name:  `<instance_name>.opstrace.io`.
+Once chosen, an instance name cannot be changed.
+
 ## Tenants
```

Comment:
Didn't we want to create a new page for that?

Reply:
Yes, just as we discussed offline.  This PR is already rather large and I want to get it merged.